src/java/edu/ucsd/library/dams/api/DAMSAPIServlet.java

```diff
@@ -5350,6 +5339,224 @@ private void loadDerivativesConfig (String derList, Map<String, String> derivati
+			addProperty(props, "fs.usemap.tgz", varNamePrefix);
+			addProperty(props, "fs.usemap.tar.gz", varNamePrefix);
+			addProperty(props, "fs.usemap.zip", varNamePrefix);
+			
+			//#ldap authorization
+			addProperty(props, "ldap.url", varNamePrefix);
+			addProperty(props, "ldap.principal", varNamePrefix);
+			addProperty(props, "ldap.pass", varNamePrefix);
+			addProperty(props, "ldap.queryPrefix", varNamePrefix);
+			addProperty(props, "ldap.querySuffix", varNamePrefix);
+			addProperty(props, "ldap.defaultAttributes", varNamePrefix);
+			addProperty(props, "ldap.memberOf.match", varNamePrefix);
+			addProperty(props, "ldap.memberOf.trimstart", varNamePrefix);
+			addProperty(props, "ldap.memberOf.trimend", varNamePrefix);
+			
+			//#fedora compat config
+			addProperty(props, "fedora.objectDS", varNamePrefix);
+			addProperty(props, "fedora.rightsDS", varNamePrefix);
+			addProperty(props, "fedora.linksDS", varNamePrefix);
+			addProperty(props, "fedora.systemDS", varNamePrefix);
+			addProperty(props, "fedora.samplePID", varNamePrefix);
+			addProperty(props, "fedora.compatVersion", varNamePrefix);
+			addProperty(props, "fedora.adminEmail", varNamePrefix);
+			//#update test
+			
+			//#activemq for solrizerd
+			addProperty(props, "queue.url", varNamePrefix);
+			addProperty(props, "queue.name", varNamePrefix);
+			
+			//#ezid for doi minting
+			addProperty(props, "ezid.target.url", varNamePrefix);
+			addProperty(props, "ezid.host", varNamePrefix);
+			addProperty(props, "ezid.shoulder", varNamePrefix);
+			addProperty(props, "ezid.user", varNamePrefix);
+			addProperty(props, "ezid.pass", varNamePrefix);
+			
+			//#FFMPEG codec parameters
+			addProperty(props, "ffmpeg.codec.params", varNamePrefix);
+		}
+		return props;
+	}
+
+	private static void addProperty(Properties props, String propName, String prefix) {
+		// Env variable convenient: APPS_[JIRA code]{deploy][purpose] APPS_DAMS_PROD_DERIVATIVES_2_RESOLUTION
+		String val = System.getProperty(prefix + "_" + propName.toUpperCase().replace("\\.", "_"));
+		props.put(propName, val);
+	}
+
+	private static String getEnvCode() throws Exception
+	{
+		String envCode = System.getProperty("RAILS_ENV");
+		if (StringUtils.isNotBlank(envCode))
+			return envCode;
+
+		List<String> envCodes = new ArrayList<>();
+		Map<String, String> vars = System.getenv();
+		for (String key : vars.keySet())
+		{
+			if (key != null && key.startsWith("APPS_DAMS_"))
+			{
+				String[] tokens = key.split("_");
+				if ( tokens.length >= 4 && !envCodes.contains(tokens[2]))
+					envCodes.add(tokens[2]);
+			}
+		}
+		if (envCodes.size() == 0)
+			return null;
+		else if (envCodes.size() == 1)
+			return envCodes.get(0);
+		else
+			throw new Exception ("Configuration error: more than one sets of environment variable found - " + envCodes.get(0) + ", " + envCodes.get(1) + ".");
+	}
+
+	private static Properties loadDefaultConfig() throws NamingException, FileNotFoundException, IOException {
```

Comment:
Yes. I think so. But I think any environment that we set up for development purpose will be the same.